study.md

```diff
@@ -40,14 +40,14 @@ What does an exclamation point after a method usually signify?  e.g.,
 `my_string.capitalize!`
 
 ```md
-<!-- your answer here -->
+<!--It will only return an object if there was a change to the object made. Otherwise it will only return nil. -->
```

Comment:
`!` in method names signifies that it will modify the object that it was invoked upon rather than return a copy of it.